pinecone/control/pinecone.py

```diff
@@ -39,7 +40,8 @@ def __init__(
         if index_api:
             self.index_api = index_api
         else:
-            api_client = ApiClient(configuration=self.config.openapi_config)
+            self.pool_threads = pool_threads
+            api_client = ApiClient(configuration=self.config.openapi_config, pool_threads=pool_threads)
```

Comment:
tiny nit, probably irrelevant but should we pass in the class instance here similar to `configuration`?

```suggestion
            api_client = ApiClient(configuration=self.config.openapi_config, pool_threads=self.pool_threads)
```